hs_core/templates/search/search.html

```diff
@@ -0,0 +1,204 @@
+         });
+         var i;
+         for (i = 1; i < div_ordering.length; i++) {
+             var div0 = "#" + div_ordering[i-1];
+             var div1 = "#" + div_ordering[i];
+             $(div1).insertAfter(div0);
+         }
+    }
+
+    var clearAllFacted = function(){
+        window.localStorage.clear();
+        var textSearch = $("#id_q").val();
+        var clearURL = "/search/?q=" + textSearch;
+        window.location = clearURL;
+    }
+
+     $(document).ready(function () {
+         $("#id_q").attr('placeholder', 'Search...');
+         reorderDivs();
+         $('.collapse').on('shown.bs.collapse', function(){
+            $(this).parent().find(".glyphicon-plus").removeClass("glyphicon-plus").addClass("glyphicon-minus");
+         }).on('hidden.bs.collapse', function(){
+             $(this).parent().find(".glyphicon-minus").removeClass("glyphicon-minus").addClass("glyphicon-plus");
+         });
+
+         $("#id_q").keypress(function(event) {
+            if (event.which == 13) {
+                event.preventDefault();
+                $("form").submit();
+            }
+         });
+         $(function () {
+             $(".faceted-selections").click(function () {
+                console.log($(this).val());
+                var tokens = $(this).val().split(",");
+                var key = tokens[0];
+                var value = tokens[1];
+                var requestURL = "{{ request.get_full_path }}"
+                if($(this).is(":checked")) {
+                    window.location = requestURL + "&selected_facets="+key+"_exact:"+value;
+                    //console.log(solr);
+                    localStorage[$(this).attr("id")] = this.checked;
+                }else{
+                    localStorage.removeItem($(this).attr("id"));
+                    var textSearch = $("#id_q").val();
+                    var updateURL = "/search/?q=" + textSearch;
+                     $(".faceted-selections").each(function () {
+                         var val = localStorage[$(this).attr("id")];
+                         var isChecked = val !== undefined ? val == "true" : false;
+                         if(isChecked){
+                             var arr = $(this).val().split(",");
+                             var key = arr[0];
+                             var value = arr[1];
+                             updateURL += "&selected_facets="+key+"_exact:"+value;
+                         }
+                     });
+                    window.location = updateURL;
+                }
+            });
+            $(".faceted-selections").each(function () {
+                var val = localStorage[$(this).attr("id")];
+                var isChecked = val !== undefined ? val == "true" : false;
+                $(this).prop("checked", isChecked);
+            });
+        });
+        $("#faceted-search").click(function(){
+            //localStorage = [];
+            window.localStorage.clear();
```

Comment:
This will clear any `localStorage` items used by *any* page on the site, not just search. You'll need to find a way to either keep everything in one item or possibly give them all a unique prefix so you can identify and remove only your `localStorage` items.